.github/workflows/docs.yml

```diff
@@ -35,12 +35,25 @@ jobs:
     steps:
       - uses: actions/checkout@11bd71901bbe5b1630ceea73d27597364c9af683 # v4.2.2
       - uses: ./.github/actions/setup-node
+      - uses: ./.github/actions/setup-python
       - name: Install dependencies
         run: |
+          pip install -r ../requirements/doc-requirements.txt
           yarn install --immutable
       - name: Run lint
         run: |
           yarn eslint
       - name: Run prettier
         run: |
           yarn prettier:check
+      - name: Build docs
```

Comment:
can create a separate job for building docs?

Reply:
Do we need `-docs`?

Reply:
Wouldn't it cause confusion if we dropped `-docs`, since this is the main mlflow repo?

Reply:
This is what it looks like:

<img width="287" alt="Image" src="https://github.com/user-attachments/assets/685e89dc-8c95-448a-a5ce-1628ed7ce773" />

It's `docs / docs` now. I think `docs / check|build|preview` is not confusing.

Reply:
Aha, got it. I was confused with the terminology, mixed up workflows and jobs. To confirm the plan is this:

- Workflow: docs.yml
  - Job: check
  - Job: build
  - Job: preview

Does that sound good? I'll move `preview-docs.yml` into `docs.yml`.